### DISCUSSION TITLE
Table Pagination

### DISCUSSION THREAD

**Discussion by ceyerkes** [twilio-labs/paste#2327] [Q&A] [answered]
Hi! Wondering what best practices are for adding pagination to a table with the new filter pattern. I've seen table pagination added as a separate filter, but wondering if that's still the preferred method, or if there's something I should be adding via the footer? Thanks!

**Comment by shleewhite** [comment, marked as answer]
Hey @ceyerkes

For the actual pagination buttons, they should stay below the table like this example: https://paste.twilio.design/components/pagination#positioning-pagination

If you're referring to a select that shows how many rows to display in the table, it can go next to the table action buttons in the filter group. In this example, it would go next to the "Export to CSV" button: https://paste.twilio.design/patterns/filter-group#default-state.

Please let me know if you have more questions!

**Reply by ceyerkes** [reply]
Thanks Lee!
